src/Makevars.win

```diff
@@ -1,6 +1,19 @@
-# Makevars.win can be tested by uploading package to win-builder.r-project.org
+VERSION=4.4.1.1
+PKG_CPPFLAGS = -I../windows/netcdf-${VERSION}/include \
+	-DHAVE_LIBUDUNITS2 -DHAVE_DECL_NC_RENAME_GRP=1
 
-PKG_CPPFLAGS = -DHAVE_LIBUDUNITS2 -DHAVE_DECL_NC_RENAME_GRP=1 \
-  -Drestrict=restrict
```

Comment:
It mainly serves as a reminder that the `restrict` keyword is being used, and that a different keyword may be needed by some compilers. This is useful on systems with older C compilers, but I suppose we don't care about them when building for Windows.